src/app/tokens.scss

```diff
@@ -197,8 +197,8 @@ $text-title-2xs: 700 24px / 26px "Metropolis", sans-serif;
 $text-title-3xs: 700 18px / 20px "Metropolis", sans-serif;
```

Comment:
Re: no longer importing metropolis.scss on L1 above, do we still want to call it on L192-197 here? Or is it already being imported elsewhere?

```diff
-@import "./fonts/metropolis.scss";
+// @import "./fonts/metropolis.scss";
```